src/Compilers/Core/Portable/CodeAnalysisEventSource.Compiler.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.CodeAnalysis
+{
+    [EventSource(Name = "Microsoft-CodeAnalysis-General")]
+    internal sealed partial class CodeAnalysisEventSource
+    {
+        public static readonly CodeAnalysisEventSource Log = new CodeAnalysisEventSource();
```

Comment:
The goal of putting this member here is to make it so you can't log events at all if you forget to declare a partial which specifies the event source name.